src/lib/holocene/link.svelte

```diff
@@ -12,6 +12,7 @@
   type $$Props = HTMLAnchorAttributes & {
     href: string;
     active?: boolean;
+    interactive?: boolean;
```

Comment:
Do I love this? No. But it's the least worst option

Reply:
Looks like link takes a `class` prop, can you just specify the styles from the parent without using a prop?

```
<Link
      href={routeForEventHistory({
        namespace,
        workflow: workflow.id,
        run: workflow.runId,
      })}
      class={isActive ? 'text-white hover:text-indigo-200 focus-visible:text-indigo-200' : ''}
    >
      {workflow.id}
  </Link>
  ```

Reply:
That doesn't work because it gets merged with text styles that are the defaults and the defaults override

Reply:
Gotcha, you would just need to inline the text styles then and use `twMerge`.